### ISSUE TITLE
API endpoint to query uploads count

### ISSUE THREAD

**Issue by farahats9** [web3-storage/web3.storage#965]
Can we have an API endpoint that returns the uploads count and the total size of the uploads?

**Comment by olizilla** [comment]
Yes! We need to prioritise this work against other fixes and features, so if you're reading this issue and you really want this api to exist too, please add a +1 on the issue description.

In the meantime, if anyone is interested in hacking on web3.storage and fancies landing a PR to add this, I will gladly help them

**Comment by dchoi27** [comment]
@mbommerez do you know if in the account limits work there's an API for this we can just expose to the user?
